### ISSUE TITLE
[Action Required] AWS Lambda end of support for Node.js 12

### ISSUE THREAD

**Issue by thohoj** [csuermann/virtual-smart-home#5]
Is there a step-by step tutorial how to to upgrade an existing backend to node 16?

Amazon Web Services wrote:
> We are ending support for Node.js 12 in AWS Lambda. This follows Node.js 12 End-Of-Life (EOL) reached on April 30, 2022 [1].
> 
> As described in the Lambda runtime support policy [2], end of support for language runtimes in Lambda happens in two stages. Starting November 14, 2022, Lambda will no longer apply security patches and other updates to the Node.js 12 runtime used by Lambda functions, and functions using Node.js 12 will no longer be eligible for technical support. In addition, you will no longer be able to create new Lambda functions using the Node.js 12 runtime. Starting December 14, 2022, you will no longer be able to update existing functions using the Node.js 12 runtime.
> 
> We recommend that you upgrade your existing Node.js 12 functions to Node.js 16 before November 14, 2022.

**Comment by thohoj** [comment]
After some research I did the following:

Edit `serverless.yml`, change 
```
  runtime: nodejs12.x
```
to
```
  runtime: nodejs16.x
```
Repeat the steps from the readme:
1. `npm install'
1. `npm install -g serverless`
1. `serverless deploy`

I checked the endpoints and arn afterwards ...they were unchanged. 

Check the lambda functions:
https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/functions

In the column "Runtime" there should be "Node.js 16.x" for each function.
